### PR TITLE
Fix tizen2016 compatibility

### PR DESCRIPTION
**Pull request by bitmovin-kenny**
- Make js code (Buffer config) ES5 compliant
- Enable DRM privilage
- Use playready DRM asset over widevine
- Enable BaseMediaDecodeTime handling explicitly
- Remove outdated explicit setting for preferredTech (that bug has been fixed)

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -21,35 +21,31 @@ function setupPlayer() {
...
+	bufferLevels[bitmovin.player.core.BufferType.ForwardDuration] = 30;
+	bufferLevels[bitmovin.player.core.BufferType.BackwardDuration] = 10,
+	bufferConfig[bitmovin.player.core.MediaType.Video] = bufferLevels;
+	bufferConfig[bitmovin.player.core.MediaType.Audio] = bufferLevels;
+	
 	var conf = {
 		key : "YOUR_PLAYER_KEY",
 		playback : {
 			autoplay : true,
-			preferredTech: [
```
Comment (mckennajones) [inline js/main.js:28]: This section was just added by me in a PR. We have seen some devices try to use the native player if this is not specified, so it should be added back.

**Review by mckennajones — CHANGES_REQUESTED** [review]
Looks good to me except the deletion below. I will verify that playback works with these changes when I'm in the office later today.

**Comment by mckennajones** [comment]
@bitmovin-kenny can confirm playback works on our 2016 device with these changes.

File: js/main.js (hunk 1 above)
Comment (bitmovin-kenny) [reply js/main.js:28]: We recently had a player bug, where Tizen (Webkit based) was detected as Safari, thus the Native route was used. This issue has been resolved, deeming this code unnecessary.

It's not wrong, it's just not needed anymore, so I thought it was a good idea to remove redundant code.

Comment (mckennajones) [reply js/main.js:28]: Thanks for clarifying @bitmovin-kenny.

**Review by mckennajones — APPROVED** [review]

**Pull request merged**